projects/go-lib/src/lib/components/go-modal/go-modal.component.ts

```diff
@@ -73,8 +88,14 @@ export class GoModalComponent implements OnInit {
       this.modalSize = this.defaultModalSize;
     }
 
+    // Set close with backdrop if provided
+    this.closeWithBackdrop = componentRef.instance['closeWithBackdrop'] === true ? true : false;
```

Comment:
Yeah, because I think technically you can pass whatever in for a value for `closeWithBackdrop` so this checks for strictly `true` instead of another value.